2019Labs/CustomSecurityContent/setup/data/ospp.profile

```diff
@@ -0,0 +1,851 @@
+documentation_complete: true
```

Comment:
For future reference, it may be good to note from which HEAD this profile came from, other then just PR description.